python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -967,6 +993,117 @@ def ready_to_take_data(self):
             )
         return self._ready_to_take_data_future
 
+    def azel_from_radec(self, ra, dec, time=None):
+        """Calculate Az/El coordinates from RA/Dec in ICRS.
+
+        Parameters
+        ----------
+        ra : `float`, `str` or `astropy.coordinates.Angle`
+            Target RA, either as a float (hour), a sexagesimal string
+            (HH:MM:SS.S or HH MM SS.S) coordinates or
+            `astropy.coordinates.Angle`.
+        dec : `float`, `str` or `astropy.coordinates.Angle`
+            Target Dec, either as a float (deg), a sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or
+            `astropy.coordinates.Angle`.
+        time : `astropy.time.core.Time` or `None`, optional
+            The time which the coordinate trasformation is intended for. If
+            `None` (default) use current time.
+
+        Returns
+        -------
+        azel : `astropy.coordinates.AltAz`
+            Astropy coordinates with azimuth and elevation.
+        """
+        radec_icrs = ICRS(Angle(ra, unit=u.hourangle), Angle(dec, unit=u.deg))
+
+        if time is None:
+            time = salobj.astropy_time_from_tai_unix(salobj.current_tai())
+
+        time.location = self.location
+
+        coord_frame_azel = AltAz(location=self.location, obstime=time)
+
+        azel = radec_icrs.transform_to(coord_frame_azel)
+
+        return azel
+
+    def radec_from_azel(self, az, el, time=None):
+        """Calculate Ra/Dec in ICRS coordinates from Az/El.
+
+        Parameters
+        ----------
+        ra : `float`, `str` or `astropy.coordinates.Angle`
+            Target RA, either as a float (hour), a sexagesimal string
+            (HH:MM:SS.S or HH MM SS.S) coordinates or
+            `astropy.coordinates.Angle`.
+        dec : `float`, `str` or `astropy.coordinates.Angle`
+            Target Dec, either as a float (deg), a sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or
+            `astropy.coordinates.Angle`.
```

Comment:
The coordinates are called az and el and not ra and dec. Please update the doc strings.

Reply:
bad copy and pasting... hehe